pages/team.vue

```diff
@@ -71,13 +153,28 @@ import ResizeImg from '@/components/widgets/ResizeImg.vue'
 
 export default {
   components: { PageLoader, ResizeImg },
+  data() {
+    return {
+      dialog: false,
+      selectedMember: {
+        name: null,
+        profile_info: null,
+        thumbnail: null,
+        uid: null
+      }
```

Comment:
Why this and not null?

Reply:
Can't set selectMember to null as the uid is needed for conditional rendering. Instead can set it to an empty object